metricbeat/module/redis/redis.go

```diff
@@ -173,28 +173,17 @@ func Select(c rd.Conn, keyspace uint) error {
 
 // CreatePool creates a redis connection pool
 func CreatePool(
-	host, password, network string,
+	uri string,
+	password string,
+	database int,
 	maxConn int,
-	idleTimeout, connTimeout time.Duration,
+	idleTimeout time.Duration,
 ) *rd.Pool {
 	return &rd.Pool{
 		MaxIdle:     maxConn,
 		IdleTimeout: idleTimeout,
 		Dial: func() (rd.Conn, error) {
-			c, err := rd.Dial(network, host,
-				rd.DialConnectTimeout(connTimeout),
-				rd.DialReadTimeout(connTimeout),
-				rd.DialWriteTimeout(connTimeout))
```

Comment:
Shouldn't we keep these options for timeouts?

Reply:
Good catch, not sure why it's deleted!